test/cljs/status_im/test/data_store/messages.cljs

```diff
@@ -6,60 +6,43 @@
 (def chat-id "chat-id")
 (def from "0x0424a68f89ba5fcd5e0640c1e1f591d561fa4125ca4e2a43592bc4123eca10ce064e522c254bb83079ba404327f6eafc01ec90a1444331fe769d3f3a7f90b0dde1")
 
-(deftest message->rpc
-  (testing "message to rpc"
-    (let [message  {:message-id message-id
-                    :content {:chat-id chat-id
-                              :response-to "id-2"
-                              :text "hta"}
-                    :whisper-timestamp 1
-                    :dedup-id "ATIwMTkwODE0YTdkNWZhZGY1N2E0ZDU3MzUxZmJkNDZkZGM1ZTU4ZjRlYzUyYWYyMDA5NTc2NWYyYmIxOTQ2OTM3NGUwNjdiMvEpTIGEjHOTAyqsrN39wST4npnSAv1AR8jJWeubanjkoGIyJooD5RVRnx6ZMt+/JzBOD2hoZzlHQWA0bC6XbdU="
-                    :outgoing-status :sending
-                    :message-type :public-group-user-message
-                    :clock-value 2
-                    :from from
-                    :chat-id chat-id
-                    :content-type "text/plain"
-                    :timestamp 3}
-          expected {:id message-id
-                    :whisperTimestamp 1
-                    :from from
-                    :chatId chat-id
-                    :replyTo "id-2"
-                    :content "{\"chat-id\":\"chat-id\",\"response-to\":\"id-2\",\"text\":\"hta\"}"
-                    :contentType "text/plain"
-                    :messageType "public-group-user-message"
-                    :clockValue 2
-                    :timestamp 3
-                    :outgoingStatus "sending"}]
-      (is (= expected (m/->rpc message))))))
-
 (deftest message<-rpc
   (testing "message to rpc"
     (let [expected  {:message-id message-id
                      :content {:chat-id chat-id
-                               :text "hta"}
+                               :sticker {:hash "hash" :pack 1}
+                               :text "hta"
+                               :line-count 1
+                               :parsed-text "parsed-text"
+                               :rtl false
+                               :response-to "a"}
                      :whisper-timestamp 1
                      :outgoing-status :sending
-                     :outgoing :sending
-                     :message-type :public-group-user-message
+                     :outgoing true
+                     :message-type 0
```

Comment:
maybe we could turn this into a keyword during the unmarshalling instead of using a constant?

Reply:
I kind of like using constants, it's a bit of nightmare to go around changing up things when the value is used directly (and prone to errors, I think we have a wrong keyword somewhere)